docs/shortcuts.md

```diff
@@ -0,0 +1,9 @@
+# Keyboard Shortcuts
+
+- <kbd>Ctrl</kbd>+<kbd>m</kbd> - toggle markdown
```

Comment:
these are actually Control or Command depending on whether you're not on a Mac

Reply:
all of them or just this one?

Reply:
Better?